### PR TITLE
Support both @returns and its alias @return as described in JSDoc 3

### PR DESCRIPTION
**Pull request by moberghammer**

### CODE REVIEW

**Comment by moberghammer** [comment]
I signed it!

2016-05-17 9:59 GMT+02:00 googlebot <notifications@github.com>:

File: closure_linter/javascriptlintrules.py [hunk 1]
```diff
@@ -287,9 +287,9 @@ def CheckToken(self, token, state):
           elif (not function.has_return and
                 not function.has_throw and
                 function.doc and
-                function.doc.HasFlag('return') and
+                (function.doc.HasFlag('returns') or function.doc.HasFlag('return')) and
                 not state.InInterfaceMethod()):
-            flag = function.doc.GetFlag('return')
+            flag = function.doc.GetFlag('returns') if function.doc.GetFlag('returns') else function.doc.GetFlag('return')
```
Comment (rumpeltux) [inline closure_linter/javascriptlintrules.py]: You can write this more idiomatic as:
flag = function.doc.GetFlag('return') or function.doc.GetFlag('returns')

**Comment by rumpeltux** [comment]
Hi, Thanks for contribution.

Please make sure that:
• all occurrences of 'return' are fixed
• all tests pass
• there is a testcase for the @returns case
• all changed code still adheres to the Google Python Styleguide: https://google.github.io/styleguide/pyguide.html

**Comment by moberghammer** [comment]
Currently python closure_linter/full_test.py run on google/closure-linter results in:
..........F................................
======================================================================
FAIL: runTest (closure_linter.common.filetestcase.AnnotatedFileTestCase)
Run linter on closure_linter/testdata/jsdoc.js
----------------------------------------------------------------------
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/closure_linter/common/filetestcase.py", line 80, in runTest
    self.assertEqual(expected, got)
AssertionError: Lists differ: [(20, 1), (20, 1), (21, 2), (2... != [(20, 1), (20, 1), (21, 2), (2...

First differing element 7:
(32, 200)
(31, 200)

Second list contains 2 additional elements.
First extra element 96:
(1324, 220)

Diff is 1497 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
Ran 43 tests in 1.392s

FAILED (failures=1)

**Pull request closed without merging**